eventtracking/backends/event_bus.py

```diff
@@ -0,0 +1,53 @@
+"""Event tracker backend that emits events to the event-bus."""
+import json
+import logging
+from datetime import datetime
+
+from openedx_events.analytics.data import TrackingLogData
+from openedx_events.analytics.signals import TRACKING_EVENT_EMITTED
+from openedx_events.event_bus import get_producer
+
+from eventtracking.backends.routing import RoutingBackend
+from eventtracking.config import SEND_TRACKING_EVENT_EMITTED_SIGNAL
+
+logger = logging.getLogger(__name__)
+
+
+class EventBusRoutingBackend(RoutingBackend):
+    """
+    Event tracker backend for the event bus.
+    """
+
+    def send(self, event):
+        """
+        Send the tracking log event to the event bus by emitting the
+        TRACKING_EVENT_EMITTED signal using custom metadata.
+        """
+        if not SEND_TRACKING_EVENT_EMITTED_SIGNAL.is_enabled():
+            return
+
+        data = json.dumps(event.get("data"))
```

Comment:
Have you tried this with a wide variety of events? I'm concerned here about possible `datetime` or other complex objects in `data` and `context` that might need to be handled.

Reply:
I think data is not always a json object so we need to cover those cases, but context is always json. For the datetime there are two possible options, it's currently generated by the tracker, so it's `datetime.now()` or it's been replayed from `event-routing-backends` and it's an str representing an UTC date. The timestamp is correctly handled, but not sure what to do with the `data` field. Can you confirm if `data` is always a json?

Reply:
So I think `data` is just more dict, but can have DateTimes in it. When serializing for the logger backend it uses the [special encoder](https://github.com/openedx/event-tracking/blob/72f6a716dc3affe3378920fa9f4439bc3112c892/eventtracking/backends/logger.py#L44) to [handle DateTimes](https://github.com/openedx/event-tracking/blob/72f6a716dc3affe3378920fa9f4439bc3112c892/eventtracking/backends/logger.py#L36), we should do the same. I don't think we need to handle it specially beyond that, since I think we want to serialize the same JSON as the log files do.

Reply:
I will keep that in mind for the next iteration on this PR